okta/src/main/java/org/ccci/idm/user/okta/dao/OktaUserDao.kt

```diff
@@ -308,6 +309,8 @@ class OktaUserDao(private val okta: Client, private val listeners: List<Listener
             grMasterPersonId = profile.getString(PROFILE_GR_MASTER_PERSON_ID)
             grPersonId = profile.getString(PROFILE_GR_PERSON_ID)
 
+            loginTime = lastLogin?.let { Instant(it.time) }
```

Comment:
Where is `lastLogin` coming from?

Reply:
an overloaded receiver. if I was explicit it would be: `this@asIdmUser.lastLogin`.

the one drawback to using `apply` is if you aren't careful you start getting overloaded receivers and it becomes unclear which object things are actually executing on.

I probably should have utilized `also` for this extension method

Reply:
I'm going to convert it to `.also` to make it more clear what is going on